docs/releases/upgrading.rst

```diff
@@ -10,9 +10,11 @@ New feature releases of Wagtail are released approximately every two months. The
 
 Additionally, patch releases will be issued as needed, to fix bugs and security issues. These are marked by incrementing the third part of the version number (for example, 1.12 to 1.12.1). Wherever possible, these releases will remain fully backwards compatible with the corresponding feature and not introduce any breaking changes.
 
-A feature release will usually stop receiving patch release updates when the next feature release comes out. However, selected feature releases are designated as Long Term Support (LTS) releases, and will continue to receive maintenance updates to address any security and data-loss related issues that arise, up until the next Long Term Support release. Typically, a Long Term Support release will happen once every four feature releases, giving a support period of eight months.
```

Comment:
I dropped the `up until the next Long Term Support release` because it would become `up until the feature release following the next Long Term Support release` which is quite confusing. The next sentence clarifies the support period anyway.